README.md

```diff
@@ -55,6 +55,33 @@ Documentation will be generated in target/doc/...
 Most documentation is taken from NaCl, with minor modification where the API
 differs between the C and Rust versions.
 
+#### Building and Testing with Support for AES
+
+The AES AEAD variant `crypto_aead_aes256gcm` requires hardware support for the
+`AES` and `CLMUL` instruction set extensions to x86; you can read why that's the
+case
+[here](https://libsodium.gitbook.io/doc/secret-key_cryptography/aead/aes-256-gcm#limitations). These instruction set extensions were first made
```

Comment:
Maybe prefer the link that includes the navigation for context, e.g. https://doc.libsodium.org/secret-key_cryptography/aead/aes-256-gcm#limitations